docs/develop/go/test-suites.mdx

```diff
@@ -160,6 +161,109 @@ When an Activity sends a Heartbeat, be sure that you can see the Heartbeats in y
 
 If an Activity is supposed to react to a Cancellation, you can test whether it reacts correctly by canceling it.
 
+## Mock and override Nexus operations
+
+Mocking Nexus operations lets you test a Workflow that executes Nexus operations without needing a
+Nexus handler servier to run the actual Nexus operation. You can mock the Nexus operation or
+override its implementation with the Nexus mock environment.
```

Comment:
```suggestion
override its implementation with the test workflow environment.
```

Reply:
Will do a pass later to do things like put sentences on separate lines and uppercase Workflow.

Reply:
Also: "test Workflow" or "Workflow test"? Does the latter make more sense?

Reply:
It's called TestWorkflowEnvironment in the code.